android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UrlDeviceDiscoveryService.java

```diff
@@ -324,9 +325,8 @@ private void updateNotifications() {
     if (!mCanUpdateNotifications) {
       return;
     }
-
     List<PwPair> pwPairs = mPwCollection.getGroupedPwPairsSortedByRank(
-        Utils.newDistanceComparator());
+        new Utils.FavoritesComparator(new HashSet<String>()));
```

Comment:
Instead of passing an empty favorites list you could just use Utils.DistanceComparator